external/llvm-project/mlir/include/mlir/Dialect/GPU/IR/CompilationAttrInterfaces.td

```diff
@@ -47,7 +47,8 @@ def GPUTargetAttrInterface : AttrInterface<"TargetAttrInterface"> {
         meant to be used for passing additional options that are not in the
         attribute.
       }], "::mlir::Attribute", "createObject",
-        (ins "const ::llvm::SmallVector<char, 0>&":$object,
+        (ins "::mlir::Operation*":$module,
```

Comment:
I think this is the only subtree-pull conflict where I needed to take our side instead of the incoming side.  All the uses of this function merged cleanly.

Reply:
Check if this hasn't been updated upstream recently - this looks like a @fabianmcg in-progress PR.

(Or, also, @fabianmcg , please advice if we should take upstream's side of this)